sdk/attestation/azure-security-attestation/tests/preparers.py

```diff
@@ -1,40 +1,113 @@
 
+from azure.security.attestation import AttestationType
+
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
+
+if TYPE_CHECKING:
+    from typing import Callable, Dict, Optional, Any, TypeVar
+
+    T = TypeVar("T")
+
 
 AttestationPreparer = functools.partial(
-            PowerShellPreparer, "attestation",
-#            attestation_azure_authority_host='xxx',
-#            attestation_resource_group='yyyy',
-#            attestation_subscription_id='xxx',
-#            attestation_environment='AzureCloud',
-            attestation_policy_signing_key0='keyvalue',
-            attestation_policy_signing_key1='keyvalue',
-            attestation_policy_signing_key2='keyvalue',
-            attestation_policy_signing_certificate0='more junk',
-            attestation_policy_signing_certificate1='more junk',
-            attestation_policy_signing_certificate2='more junk',
-            attestation_serialized_policy_signing_key0="junk",
-            attestation_serialized_policy_signing_key1="junk",
-            attestation_serialized_policy_signing_key2="junk",
-            attestation_serialized_isolated_signing_key='yyyy',
-            attestation_isolated_signing_key='xxxx',
-            attestation_isolated_signing_certificate='xxxx',
-            attestation_service_management_url='https://management.core.windows.net/',
-            attestation_location_short_name='wus', # Note: This must match the short name in the fake resources.
-            attestation_client_id='xxxx',
-            attestation_client_secret='secret',
-            attestation_tenant_id='tenant',
-            attestation_isolated_url='https://fakeresource.wus.attest.azure.net',
-            attestation_aad_url='https://fakeresource.wus.attest.azure.net',
-#            attestation_resource_manager_url='https://resourcemanager/zzz'
-        )
+    PowerShellPreparer,
+    "attestation",
+    #            attestation_azure_authority_host='xxx',
+    #            attestation_resource_group='yyyy',
+    #            attestation_subscription_id='xxx',
+    #            attestation_environment='AzureCloud',
+    attestation_policy_signing_key0="keyvalue",
+    attestation_policy_signing_key1="keyvalue",
+    attestation_policy_signing_key2="keyvalue",
+    attestation_policy_signing_certificate0="more junk",
+    attestation_policy_signing_certificate1="more junk",
+    attestation_policy_signing_certificate2="more junk",
+    attestation_serialized_policy_signing_key0="junk",
+    attestation_serialized_policy_signing_key1="junk",
+    attestation_serialized_policy_signing_key2="junk",
+    attestation_serialized_isolated_signing_key="yyyy",
+    attestation_isolated_signing_key="xxxx",
+    attestation_isolated_signing_certificate="xxxx",
+    attestation_service_management_url="https://management.core.windows.net/",
+    attestation_location_short_name="wus",  # Note: This must match the short name in the fake resources.
```

Comment:
You're replacing the region with "wus" here

Reply:
I think you can remove this line entirely.

Reply:
??? I'm not sure I can - this is a variable output from the new-test-resources.ps1 script and indicates the short region name. It's needed for the tests of the shared attestation resource.